mender-convert

```diff
@@ -343,22 +363,24 @@ do_make_sdimg_beaglebone() {
   local ret=0
 
   create_device_maps $raw_disk_image raw_disk_mappings
-
-  mount_mender_disk ${mender_disk_mappings[@]}
   mount_raw_disk ${raw_disk_mappings[@]}
 
   log "$step/$total Setting boot partition..."
   ((step++))
-  stage_2_args="$sdimg_boot_dir $embedded_rootfs_dir"
+
+  stage_2_args="$output_dir ${mender_disk_mappings[0]} ${embedded_rootfs_dir}"
   ${tool_dir}/bbb-convert-stage-2.sh ${stage_2_args} || ret=$?
   [[ $ret -ne 0 ]] && { log "Aborting."; return $ret; }
 
   log "$step/$total Setting root filesystem partition..."
   ((step++))
+
   stage_3_args="$sdimg_primary_dir $embedded_rootfs_dir"
   ${tool_dir}/bbb-convert-stage-3.sh ${stage_3_args} || ret=$?
```

Comment:
@mirzak Please use common `convert-stage-3.sh` script, as `bbb-convert-stage-3.sh` is no longer available.

Reply:
Yeah, noticed this one.